docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext",
+    "vscode": {
+     "languageId": "raw"
+    }
+   },
+   "source": [
+    ".. graphviz::\n",
+    "\n",
+    "    digraph {\n",
+    "        rankdir=LR;\n",
+    "        bgcolor=\"transparent\";\n",
+    "        node [shape=box]; // Default shape\n",
+    "        // Define nodes\n",
+    "        \"SQ-LNS\";\n",
+    "        \"stunting\";\n",
+    "        \"wasting\";\n",
+    "        \"underweight\";\n",
+    "        \"iron_deficiency\" [label=\"iron deficiency\"];\n",
+    "        \"cognitive_development\" [label=\"cognitive development\"];\n",
+    "        \"anemia\";\n",
+    "        \"mortality\";\n",
+    "\n",
+    "        // Define solid edges\n",
+    "        \"SQ-LNS\" -> \"stunting\";\n",
+    "        \"SQ-LNS\" -> \"wasting\";\n",
+    "        \"SQ-LNS\" -> \"underweight\";\n",
+    "        \"SQ-LNS\" -> \"iron_deficiency\";\n",
+    "        \"SQ-LNS\" -> \"cognitive_development\";\n",
+    "        \"iron_deficiency\" -> \"anemia\";\n",
+    "\n",
+    "        // Define dashed edges with labels\n",
+    "        \"stunting\" -> \"mortality\" [style=dashed, label=\"?\"];\n",
+    "        \"wasting\" -> \"mortality\" [style=dashed, label=\"?\"];\n",
+    "        \"underweight\" -> \"mortality\" [style=dashed, label=\"?\"];\n",
+    "        \"anemia\" -> \"mortality\" [style=dashed, label=\"?\"];\n",
+    "        \"cognitive_development\" -> \"mortality\" [style=dashed, label=\"?\"];\n",
+    "        \"SQ-LNS\" -> \"mortality\" [style=dashed, label=\"?\"];\n",
+    "    }"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now it is time to bring the [**Global Burden of Disease Study (GBD)**](https://www.healthdata.org/research-analysis/gbd) into the picture.\n",
+    "In almost every global health model we make, GBD is the cornerstone data source.\n",
+    "It provides us with estimates about a wide array of health outcomes and risk factors in\n",
+    "a standard format, which allows us to much more quickly build simulations using these data.\n",
+    "GBD estimates are the result of a very complex process; in this tutorial,\n",
+    "we won't dive any deeper into GBD methods than we absolutely need to,\n",
+    "but we will provide guidance on where you can learn more.\n",
+    "\n",
+    "Some foundational GBD terminology: **burden** (measured in disability-adjusted life years \\[DALYs\\])\n",
+    "is composed of **fatal burden** (burden due to premature death, measured in years of life lost \\[YLLs\\])\n",
+    "and **nonfatal burden** (burden due to disability, measured in years lived with disability \\[YLDs\\]).\n",
+    "In fact, DALYs equal the sum of YLLs and YLDs.\n",
+    "\n",
+    "All burden has a **cause** -- this general word means something very specific in GBD!\n",
```

Comment:
Is this the correct plurality for burden? Hah, I definitely would've said "All burdens have a cause..."

Reply:
I use burden is an uncountable noun in the GBD context 🤷